packages/core/src/postProcess/PostProcessPass.ts

```
@@ -0,0 +1,83 @@
+  get isActive(): boolean {
+    return this._isActive;
+  }
+
+  set isActive(value: boolean) {
+    if (value !== this._isActive) {
+      this._isActive = value;
+
+      if (value) {
+        this._postProcessManager && this._postProcessManager.addPostProcessPass(this);
+      } else {
+        this._postProcessManager && this._postProcessManager._removePostProcessPass(this);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)